examples/simple-nft-minter/index.mjs

```diff
@@ -0,0 +1,72 @@
+import { Reach, test } from '@reach-sh/stdlib';
+const stdlib = new Reach(process.env, { REACH_NO_WARN: 'Y' });
+
+const startingBalance = stdlib.parseCurrency(100);
+const [minter, receiver] = await stdlib.newTestAccounts(2, startingBalance);
+minter.setDebugLabel('Minter');
+receiver.setDebugLabel('Receiver');
+
+const gasLimit = 500000;
+if (stdlib.connector != 'ALGO') {
+    minter.setGasLimit(gasLimit) && receiver.setGasLimit(gasLimit)
+};
+
+const mintAddr = minter.getAddress();
+console.log(`Minter's address is ${mintAddr}`);
+const recAddr = receiver.getAddress();
+console.log(`Receiver's address is ${recAddr}`);
+
+const minterAddrFormat = await stdlib.formatAddress(minter);
+console.log(`The minter's formatted address is ${minterAddrFormat}`);
+const receiverAddrFormat = await stdlib.formatAddress(receiver);
+console.log(`The receiver's formatted address is ${receiverAddrFormat}`);
+
+const fmt = (x) => stdlib.formatCurrency(x, 4);
+const getBal = async (who, tok) => tok ? (await stdlib.balanceOf(who, tok)) : fmt(await stdlib.balanceOf(who));
+
+const logBalance = async (acc, tok) => {
+    const bal = await getBal(acc, tok);
+    const unit = tok ? 'of the NFT' : stdlib.standardUnit;
+    console.log(`${acc.getDebugLabel()} has ${bal} ${unit}.`);
+    return bal;
+}
+
+await logBalance(minter, null);
+
+const name = (stdlib.connector == 'ALGO') ? "JPAlgos" : "JPals";
+const symbol = (stdlib.connector == 'ALGO') ? "JPA" : "JPAL";
```

Comment:
I think for the purposes of this tutorial, there's no need to be so detailed regarding names. Maybe just pick a generic name and symbol?

Reply:
In an earlier review Jay asked for a cute NFT name. 
I wanted to show that the contract works on Algo or ETH.
Seems like most of the NFTs on Algo fit "algo" into the name, but that doesn't make sense for ETH deployments.

Reply:
I agree with Nevers. I think any time you write a tutorial, you want to pursue simplicity at all costs. My advice for cuteness is for the name to be "JPals" and the symbol to be "JPs" ;)